.github/workflows/pytest.yml

```diff
@@ -42,11 +42,14 @@ jobs:
 
       - name: pip installs
         run: |
-          pip install pytest pytest-cov pytest-xdist setuptools_scm
+          pip install wheel
+          pip install pytest-cov pytest-xdist setuptools_scm nbval
           pip install -e .
       - name: run pytest
         run: |
-          pytest --runslow -n2
+          pytest -n2 --dist loadscope --nbval --cov=weldx --current-env tutorials/
+          pytest --runslow -n2 --cov-append --cov=weldx tests/
```

Comment:
we do we need to set `--cov=weldx` here? It should get set from `setup.cfg`

Reply:
as far as I can see, there is no such setting in setup.cfg. It should contains this, IMO:

```
[coverage:run]
include = weldx
```

I wonder how it worked before, as pytest would have recorded the coverage for all code.

Reply:
it is included in the pytest section
https://github.com/BAMWelDX/weldx/blob/529ced9040111948e28554a0d6f54bc17b368d0b/setup.cfg#L87-L96

Reply:
OK, and nbval picks up the settings from pytest-cov. So I can omit it in the pytest invocation.